code/modules/transport/transport_module.dm

```diff
@@ -933,6 +933,6 @@
 /obj/structure/transport/linear/tram/proc/estop_throw(throw_direction)
 	for(var/mob/living/passenger in transport_contents)
 		to_chat(passenger, span_userdanger("The tram comes to a sudden, grinding stop!"))
+		var/extra_flight = prob(20) // sometimes you go through a window
 		var/throw_target = get_edge_target_turf(src, throw_direction)
-		var/datum/callback/land_slam = new(passenger, TYPE_PROC_REF(/mob/living/, tram_slam_land))
-		passenger.throw_at(throw_target, 400, 4, force = MOVE_FORCE_OVERPOWERING, callback = land_slam)
+		passenger.throw_at(throw_target, 30, 6, force = MOVE_FORCE_OVERPOWERING, smash_windows = extra_flight)
```

Comment:
We could in theory make this a lot more organic by using throw force to determine whether you break through something or not

And then each object could set a level of force required to be thrown through, IE normal windows require less force than others

Reply:
That was my first thought, I'd be up to looking at that when expanding this to things other than the tram.

Reply:
Since this is only used on the tram, it makes sense to call it from /obj/structure/transport/linear/tram instead of adding the arg to the entire throw proc.